app/helpers/budgets_helper.rb

```diff
@@ -76,4 +76,17 @@ def current_budget_map_locations
       }
     end.flatten.compact
   end
+
+  def calculate_winner_investments(budget)
```

Comment:
Helpers are awesome, we try to stick to using html in views and logic without html in the helpers
How about something like this?

#helpers
```
def display_calculate_winners_button?(budget)
  budget.balloting_or_later?
end

def calcuate_winner_button_text(budget)
  if budget.investments.winners.empty? 
    t("admin.budgets.winners.calculate")
  else
    t("admin.budgets.winners.recalculate")
  end
end
```

#view
```
<% if display_calculate_winners_button?(budget) %>
  <%= link_to button_text,
              calcuate_winner_button_text(budget),
              method: :put,
              class: "button hollow float-right" %>
<% else %>
  <span class="button hollow disabled float-right">
    <%= t("admin.budgets.winners.calculate")%>
    <div class="callout warning clear">
      <%= t("admin.budget_investments.index.cannot_calculate_winners") %>
    </div>
  </span>
<% end %>
```